src/App.tsx

```diff
@@ -1,109 +1,17 @@
 import React from 'react';
 
 import './App.scss';
+import { PostList } from './components/PostList';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
 
 export const App: React.FC = () => (
   <section className="App">
     <h1 className="App__title">Static list of posts</h1>
 
-    <div className="PostList">
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">qui est esse</h3>
+    <PostList
+      posts={postsFromServer}
+    />
```

Comment:
More readability, but it's up to you
```suggestion
    <PostList posts={postsFromServer} />
```